stage0/src/CMakeLists.txt

```diff
@@ -35,7 +35,6 @@ option(CCACHE             "use ccache"         ON)
 option(SPLIT_STACK        "SPLIT_STACK"        OFF)
 # When OFF we disable LLVM support
 option(LLVM               "LLVM"               OFF)
-
 # When ON we include githash in the version string
```

Comment:
Another unrelated whitespace change.

Reply:
Should be resolved, post `stage0` revert.